plugins/janus_videoroom.c

```diff
@@ -251,6 +251,15 @@ typedef enum janus_videoroom_videocodec {
 	JANUS_VIDEOROOM_VP9,	/* Publishers will have to use VP9 */
 	JANUS_VIDEOROOM_H264	/* Publishers will have to use H264 */
 } janus_videoroom_videocodec;
+
+typedef enum janus_videoroom_audiocodec {
+	JANUS_VIDEOROOM_OPUS,	/* Publishers will have to use OPUS */
+	JANUS_VIDEOROOM_ISAC_32K,	/* Publishers will have to use ISAC 32K */
+	JANUS_VIDEOROOM_ISAC_16K,	/* Publishers will have to use ISAC 16K */
+	JANUS_VIDEOROOM_PCMU	/* Publishers will have to use PCMU 8K */
```

Comment:
Thanks. I will add PCMA ( a-Law ), and will test, and send you another request with fixes, and suggestions that you mentioned in your inline comments. Thanks for your time.